ontology/yaml/resources/SAFETY/entity_types/GENERALTYPES.yaml

```diff
@@ -75,4 +75,10 @@ LDS:
   is_abstract: true
   implements:
   - EQUIPMENT
+  
+RDT:
```

Comment:
needs to also be submitted for approval in [BDNS](https://github.com/theodi/BDNS/blob/master/BDNS_Abbreviations_Register.csv)

Reply:
Incorporated in BDNS